mkctr.go

```diff
@@ -306,10 +318,9 @@ func fetchAndBuild(bp *buildParams) error {
 		adds = append(adds, mutate.IndexAddendum{
 			Add: img,
 			Descriptor: v1.Descriptor{
-				MediaType:   id.MediaType,
-				URLs:        id.URLs,
-				Annotations: id.Annotations,
```

Comment:
This PR also has the side effect that we no longer preserve any annotations that were set on base image. I think we shouldn't need to looking at what the typical annotation values are, they are image specific and wouldn't apply to the top level image. (The alpine base we use does not have any annotations).
(cc @maisem )